EnglishTranslation-master/UI.tsv

```diff
@@ -2816,7 +2816,7 @@ UI_20170912_002815	News	공지글
 UI_20170912_002816	Notes	소개글				
 UI_20170912_002817	Name {img %s 13 13}	길드원{img %s 13 13}				
```

Comment:
This still needs to be reverted, it has been fine for months and is not the translation nor a desired localization